package.json

```diff
@@ -6,7 +6,8 @@
   "devDependencies": {},
   "dependencies": {
     "child-process-promise": "^2.2.1",
-    "xdl": "^51.5.0",
-    "semver": "^5.6.0"
+    "detox": "^12.2.0",
```

Comment:
Maybe you can add detox as peer dependency

Reply:
Done